core/src/lib.rs

```diff
@@ -99,12 +103,22 @@ pub type DefaultFileService = FileServiceImpl<
 
 #[derive(Debug, Serialize)]
 pub enum InitLoggerError {
+    OnTouch(String),
     Unexpected(String),
 }
 
 pub fn init_logger(log_path: &Path) -> Result<(), InitLoggerError> {
-    loggers::init(log_path, env::var("LOCKBOOK_DEBUG").is_ok())
-        .map_err(|err| InitLoggerError::Unexpected(format!("{:#?}", err)))
+    let print_debug = env::var("LOCKBOOK_DEBUG").is_ok();
+    let print_colors = env::var("LOCKBOOK_NO_COLOR").is_err();
+    match loggers::init(log_path, print_debug, print_colors) {
+        Ok(_) => Ok(()),
+        Err(LoggersError::File(_)) => match create_dir_all(log_path) {
```

Comment:
I'm not sure this is the best way to handle this error, you should just check if the folder exists and create it if not.

Reply:
There is some difference in performance:
```
raayanpillai@raayan-mbp ~/dev/lockbook/clients/cli [raayan/fix_clogger] $ rm -r ~/.lockbook
raayanpillai@raayan-mbp ~/dev/lockbook/clients/cli [raayan/fix_clogger] $ time lockbook list
No account! Run init or import to get started!
lockbook list  0.01s user 0.01s system 21% cpu 0.128 total
raayanpillai@raayan-mbp ~/dev/lockbook/clients/cli [raayan/fix_clogger] $ time lockbook list
No account! Run init or import to get started!
lockbook list  0.01s user 0.01s system 47% cpu 0.054 total
```

Also I put this here https://github.com/lockbook/lockbook/pull/245/files/bc16e4ac41b8ab38793835b0773ed315d80b72f4#r476084369 for the sake of discussion, I wasn't planning to die on this hill